package.json

```diff
@@ -32,8 +32,19 @@
     "q": "^1.4.1"
   },
   "devDependencies": {
+    "eslint": "^1.10.3",
     "mocha": "^2.2.5"
   },
+  "eslintConfig": {
+    "extends": "xo-space",
+    "rules": {
+      "camelcase": 0,
```

Comment:
For `/home/zckrs/github/yeoman-generator-list/newrelic.js` just disable the rule in the file with `/* eslint-disable camelcase */` at the top.

With `/home/zckrs/github/yeoman-generator-list/plugin-cache/src/gh-stats.js` do the same, but just inline.